src/consumer/rpc/v3_3/node/node_notify.ts

```diff
@@ -0,0 +1,64 @@
+/*
+* Copyright 2021 HMS Industrial Networks AB
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http: //www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { plainToClass, Type } from 'class-transformer';
+import { IsDefined, ValidateNested, IsObject, IsNumber } from 'class-validator';
+import { KolibriRequestMethods } from '../../../kolibri_request_methods';
+import { IsKolibriNodePath } from '../../../../validation/decorators/is_kolibri_node_path';
+import { JsonRpcId } from '../../../jsonrpc';
+import { KolibriRequest } from '../../../kolibri_request';
+import { DefaultKolibriResponse } from '../../../kolibri_response';
+
+export class NodeNotifyParams {
+    @IsKolibriNodePath()
+    path: string;
+
+    @IsNumber()
+    timestamp: number;
+
+    @IsObject()
+    data: Object
```

Comment:
Since we know the possible data type. We can also create classes for them.

Reply:
Done